packages/components/src/utilities/focusStyle.ts

```diff
@@ -0,0 +1,12 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { twMerge } from 'tailwind-merge';
```

Comment:
Could you create a utility `cn` or `merge` or something else which just wraps the `twMerge` utility?
Also please add tests to it

Reply:
alias added